Update AMMCreate tx cost and amm_info syntax. Fix #44 and #49. (#58)
ripple/opensource.ripple.com: docs/xls-30d-amm/transaction-types/ammcreate.md

One or both of `Amount` and `Amount2` can be [tokens](https://xrpl.org/tokens.html); at most one of them can be [XRP](https://xrpl.org/xrp.html). They cannot both have the same currency code and issuer. The tokens' issuers must have [Default Ripple](https://xrpl.org/rippling.html#the-default-ripple-flag) enabled. If the Clawback amendment is enabled, those issuers must not have the ability to clawback their tokens. An AMM's LP tokens _can_ be used as one of the assets for another AMM.

## Special Transaction Cost

> Since each AMM instance involves an AccountRoot ledger entry, an AMM ledger entry, and one to two trust lines to hold the token or tokens in its pool, an AMMCreate transaction requires a much higher than usual [transaction cost](https://xrpl.org/transaction-cost.html) as a deterrent against ledger spam. Instead of the standard minimum of 0.00001 XRP, AMMCreate must destroy at least the incremental owner reserve amount, currently 2 XRP. This is the same special transaction cost as AccountDelete transactions.

Reviewer: ```suggestion
Since each AMM instance involves an AccountRoot ledger entry, an AMM ledger entry, and a trust line for each token in its pool, AMMCreate transactions requires a higher [transaction cost](https://xrpl.org/transaction-cost.html) to deter against ledger spam. Instead of the standard minimum of 0.00001 XRP, AMMCreate must destroy at least the incremental owner reserve amount, currently 2 XRP. This is the same special transaction cost as AccountDelete transactions.
```

Author: Went with _almost_ this wording. The above suggestion has subject-verb agreement issues with "transactions requires", so I left that part unchanged, and I changed "to deter against ledger spam" to just "to deter ledger spam".
